cs/src/core/Index/FasterLog/FasterLogSettings.cs

```diff
@@ -77,6 +77,11 @@ public class FasterLogSettings
         /// </summary>
         public LogChecksumType LogChecksum = LogChecksumType.None;
 
+        /// <summary>
+        /// Fraction of log marked as mutable (uncommitted)
+        /// </summary>
+        public double MutableFraction = 0;
```

Comment:
does this value have any effect on when commits get auto-triggered if I'm only scanning committed entries?

Reply:
The default `MutableFraction` of 0 means that every page will auto-commit as soon as it is filled up. If you have 8 pages in memory and `MutableFraction` is set to 0.5, the last 4 pages will stay mutable (uncommitted) in the absence of explicit `Commit` requests. In other words, the pages in the immutable fraction of memory are auto-committed.